### PR TITLE
Replace the only EGD call by urandom

### PR DESCRIPTION
**Pull request by hynek**
Also harmonize documentation.  #boyScout

### CODE REVIEW

**Comment by hynek** [comment]
Fixes #191

File: doc/api/rand.rst [hunk 1]
```diff
@@ -1,79 +1,43 @@
...
-.. py:module:: OpenSSL.rand
-   :synopsis: An interface to the OpenSSL pseudo random number generator
-
-
-This module handles the OpenSSL pseudo random number generator (PRNG) and
-declares the following:
-
-.. py:function:: add(string, entropy)
+.. warning::
+   Functions from this methods shouldn't be used.
```
Comment (reaperhulk) [inline doc/api/rand.rst]: s/methods/module right?

File: doc/api/rand.rst [hunk 2]
```diff
@@ -1,79 +1,43 @@
...
+   `Use urandom <http://sockpuppet.org/blog/2014/02/25/safely-generate-random-numbers/>`_ instead.
```
Comment (Lukasa) [inline doc/api/rand.rst]: Is it worth adding to the documentation what the deprecation cycle is?

File: src/OpenSSL/version.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
     "__title__", "__uri__", "__version__",
 ]
 
-__version__ = "0.16.dev0"
+__version__ = "15.2.0.dev0"
```
Comment (Lukasa) [inline src/OpenSSL/version.py]: =O Version scheme changes!

**Comment by Lukasa** [comment]
Some small notes (suggestion for more docs, notes on the version number change), otherwise I'm happy. =)

File: doc/api/rand.rst (hunk 2 above)
Comment (hynek) [inline doc/api/rand.rst]: We're not deprecating the whole module, that's just good-natured advice. :)

I wanna write a backwards compat doc before release (2 year after deprecation it goes to hell).

File: src/OpenSSL/version.py (hunk 3 above)
Comment (hynek) [inline src/OpenSSL/version.py]: Yep, I've discussed that on #cryptography-dev.   SemVer just doesn't make any sense for pyOpenSSL.

**Comment by Lukasa** [comment]
\o/

**Pull request merged**